docs/product/integrations/notification-incidents/pagerduty/index.mdx

```diff
@@ -44,11 +44,11 @@ If you re-generate an integration key for one of your services in PagerDuty you
 
 Click "Add Alert Rule" in the integration configuration page or go to **Alerts** and click "Create Alert" to set up a new alert rule for the PagerDuty integration.
 
-In [issue alerts](/product/alerts/alert-types/#issue-alerts), select "Send a PagerDuty notification" in the actions dropdown and then select your account and service:
+In [issue alerts](/product/alerts/alert-types/#issue-alerts), select "Send a PagerDuty notification" in the actions dropdown and then select your account and service. You can also select a custom severity for alert, or select "default" to use the default severity level that Sentry provides for issue types ("critical" for fatal, "error" for error, "warning" for warning, and "info" for debug and info issues).
 
 ![PagerDuty alert rule action](/product/integrations/notification-incidents/pagerduty/pd_alert_rule.png)
 
-In [metric alerts](/product/alerts/alert-types/#metric-alerts), select the PagerDuty option in the dropdown that corresponds to your account and the select your service:
+In [metric alerts](/product/alerts/alert-types/#metric-alerts), select the PagerDuty option in the dropdown that corresponds to your account and the select your service. You can also select a custom severity for the alert, or leave it blank to use the default severity level that Sentry provides for incident statuses ("critical" for critical status and "warning" for warning status).
```

Comment:
```suggestion
In [metric alerts](/product/alerts/alert-types/#metric-alerts), select the PagerDuty option in the dropdown that corresponds to your account and then select your service. You can also select a custom severity for the alert, or leave it blank to use the default severity level that Sentry provides for incident statuses ("critical" for critical status and "warning" for warning status).
```

Reply:
From the image below, it looks like you can also set an "error" status?

Reply:
critical and warning are the only kinds of metric alerts you can set. the pagerduty default severities just happen to align exactly with those